### PR TITLE
ES BulkProcessor

### PR DESCRIPTION
**Pull request by tbuchaillot**
Adding a new way of writing on ES,  ES BulkProcessor.

New config inside ES Meta:
```go
       "disable_bulk": false, 
        "bulk_config":{
            "wokers": int,             //number of wokers
            "bulk_actions":int ,   //number of requests
            "flush_interval":int,  //time in seconds
            "bulk_size": int        //ammount of bytes
        }
```

When any of these conditions are satisfied,  the batch of analytics records is sent to the ES.

Solves issue #204

### CODE REVIEW

**Comment by tbuchaillot** [comment]
WIP Readme

**Comment by buger** [comment]
Can we have some good defaults?

**Comment by tbuchaillot** [comment]
@buger I think the best idea is making it work like the ES Bulk API when none of the bulk_conf are set. 
Maybe when #203 is merged we can use the `chunkSize` like the default `bulk_actions` number?

File: pumps/elasticsearch.go [hunk 1]
```diff
@@ -62,14 +73,99 @@ func getOperator(version string, url string, setSniff bool) (ElasticsearchOperat
...
+
 		return e, err
 	case "6":
 		e := new(Elasticsearch6Operator)
 		e.esClient, err = elasticv6.NewClient(elasticv6.SetURL(urls...), elasticv6.SetSniff(setSniff))
+		if err != nil {
+			return e, err
+		}
+		// Setup a bulk processor
+		p := e.esClient.BulkProcessor().Name("TykPumpESv6BackgroundProcessor")
```
Comment (letzya) [inline pumps/elasticsearch.go]: Any chance we can avoid triplicating the code? 33

**Pull request merged**